### PR TITLE
Buildsystem cleanup

### PR DESCRIPTION
**Pull request by Alecaddd**
## Summary / How this PR fixes the problem?
Completely remove `GooCanvas` from being built, required, or used.
Update the build system and flatpak tester to run on `main`.
Fixed a couple of areas where `Lib2` was using attributes from `Lib1`.
Commented out old files from the meson build, but left the in the repo as reference.

## Steps to Test
Nothing should change in the current state of `main` and the canvas should behave as expected.

### CODE REVIEW

File: com.github.akiraux.akira.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+sdk: io.elementary.Sdk
+command: com.github.akiraux.akira
+
+finish-args:
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+
+  # needed for perfers-color-scheme
+  - '--system-talk-name=org.freedesktop.Accounts'
```
Comment (bilelmoussaoui) [inline com.github.akiraux.akira.yml:13]: Is it really needed? I thought elementary added a gsetting proxy for it.

File: aux/flatpak/com.github.akiraux.akira.json [hunk 2]
```diff
@@ -23,21 +23,7 @@
     "--filesystem=xdg-run/gvfs",
     "--filesystem=xdg-run/gvfsd"
```
Comment (bilelmoussaoui) [inline aux/flatpak/com.github.akiraux.akira.json:24]: We should drop one of the two manifests.

File: com.github.akiraux.akira.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+  - '--filesystem=home'
```
Comment (bilelmoussaoui) [inline com.github.akiraux.akira.yml:15]: Is this needed ?

File: com.github.akiraux.akira.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  - '--device=dri'
+
+modules:
+  - name: akira
+    buildsystem: meson
+    sources:
+      - type: dir
+        path: .
```
Comment (bilelmoussaoui) [inline com.github.akiraux.akira.yml:23]: That's actually incorrect. The path is ..

File: com.github.akiraux.akira.yml (hunk 1 above)
Comment (Alecaddd) [reply com.github.akiraux.akira.yml:13]: It's needed based on their documentation to detect the preferred color scheme.
Do you see any issue with this?

File: aux/flatpak/com.github.akiraux.akira.json (hunk 2 above)
Comment (Alecaddd) [reply aux/flatpak/com.github.akiraux.akira.json:24]: which one would you drop?

File: com.github.akiraux.akira.yml (hunk 3 above)
Comment (Alecaddd) [reply com.github.akiraux.akira.yml:15]: Yes, without this is not possible to save/load files from the home directory.

File: com.github.akiraux.akira.yml (hunk 4 above)
Comment (Alecaddd) [reply com.github.akiraux.akira.yml:23]: really? I added this to `master` and successful released the Akira flatpak in the elementary AppCenter.

Comment (bilelmoussaoui) [reply com.github.akiraux.akira.yml:23]: Ah nvm, reviewing from a phone is hard. I thought the file is still in aux folder.

File: com.github.akiraux.akira.yml (hunk 3 above)
Comment (bilelmoussaoui) [reply com.github.akiraux.akira.yml:15]: We should switch to using Native dialogs wherever we can I guess

File: com.github.akiraux.akira.yml (hunk 1 above)
Comment (bilelmoussaoui) [reply com.github.akiraux.akira.yml:13]: Not really no, I just thought something has changed meanwhile. I don't think it's a good idea to open a hole in the sandbox for each app just to get color scheme/dark mode working but that's not related to Akira.

File: com.github.akiraux.akira.yml (hunk 3 above)
Comment (Alecaddd) [reply com.github.akiraux.akira.yml:15]: Yeah, that's on my list.

File: src/Services/EventBus.vala [hunk 5]
```diff
@@ -64,10 +64,10 @@ public class Akira.Services.EventBus : Object {
     // Item signals.
     public signal void change_z_selected (bool raise, bool total);
     public signal void flip_item (bool vertical = false);
-    public signal void request_add_item_to_selection (Lib.Items.CanvasItem item);
-    public signal void request_delete_item (Lib.Items.CanvasItem item);
-    public signal void selected_items_changed (List<Lib.Items.CanvasItem> selected_items);
-    public signal void selected_items_list_changed (List<Lib.Items.CanvasItem> selected_items);
+    // public signal void request_add_item_to_selection (Lib.Items.CanvasItem item);
```
Comment (mbfraga) [inline src/Services/EventBus.vala:67]: Just get rid of these tbh.

File: src/Utils/AffineTransform.vala [hunk 6]
```diff
@@ -20,8 +20,8 @@
  * Authored by: Alessandro "Alecaddd" Castellani <castellani.ale@gmail.com>
  */
 
-using Akira.Lib.Items;
-using Akira.Lib.Managers;
+// using Akira.Lib.Items;
```
Comment (mbfraga) [inline src/Utils/AffineTransform.vala:23]: Nothing Lib1 related is useful in here anymore. Just remove.

**Review by mbfraga — CHANGES_REQUESTED** [review]

**Review by mbfraga — APPROVED** [review]

**Pull request merged**